testing/content/files/en-us/learn/css/css_layout/introduction/flex/index.html

```diff
@@ -16,7 +16,7 @@ <h2 id="Flexbox">Flexbox</h2>
 and also tests that prerequisites are handled as part of the rendering
 and live-sample-building process.
 -->
-<div id="Flex_1">{{ Page("Learn/CSS/CSS_layout/Introduction/Flex/Stuff", "Flex_1") }}</div>
+<section id="Flex_1">{{ Page("Learn/CSS/CSS_layout/Introduction/Flex/Stuff", "Flex_1") }}</section>
```

Comment:
@fiji-flo Do you know whether there are any occurrences of `<div id="...">` outside of code blocks in content or translated-content that might have been added intentionally and are used to link to them?

Does rari check that the targets of anchor links are actually defined? I believe yari does not, but this would probably be useful, as links to anchors that don't exist are essentially dead links.